server/utils/verifiers/VerifierBase.ts

```diff
@@ -0,0 +1,350 @@
+                    continue;
+                }
+
+                // if identifier exists in our database (value & type) then store it
+                const identifiers: DBAPI.Identifier[] | null = await DBAPI.Identifier.fetchFromIdentifierValue(content);
+                if(identifiers) {
+                    for(const identifier of identifiers) {
+                        const details: IdentifierDetails | null = await this.getIdentifierType(identifier);
+                        if(!details) {
+                            LOG.error(`could not get details for EDAN identifier (type: ${identifier.idVIdentifierType} | value:${identifier.IdentifierValue})`, LOG.LS.eTEST);
+                            continue;
+                        }
+
+                        // if we have an identifier that is of the same type then store
+                        if(details.identifierTypeEnum===identifierTypeEnum) {
+                            result.details?.push(details);
+                            break;
+                        }
+                    }
+
+                } else {
+                    // didn't find the identifier in our database so create one
+                    // TODO: make DBAPI.Identifier object
+                    const details: IdentifierDetails = { identifier: null, identifierType, identifierTypeEnum };
+                    result.details?.push(details);
+                }
+
+                // console.log('EDAN: '+label+'|'+content+'|'+JSON.stringify(vIdentifierType));
+            }
+        }
+
+        return result;
+    }
+
+    protected async replacePackratUnit(packratUnit: DBAPI.Unit | null, edanUnit: DBAPI.Unit): Promise<boolean> {
+
+        // if packrat unit is null, we need to create one and push to the system
+        if(!packratUnit) { return false; }
+        //     console.warn(edanUnit);
+
+        //     const unitArgs = {
+        //         idUnit: 0, // use Edan's?
+        //         Name: edanUnit.Name,
+        //         Abbreviation: edanUnit.Abbreviation,
+        //         ARKPrefix: edanUnit.ARKPrefix
+        //     };
+        //     packratUnit = new DBAPI.Unit(unitArgs);
+        //     const wasCreated: boolean = await packratUnit.create();
+        //     if(!wasCreated) {
+        //         LOG.error(`failed to create new unit in Packrat (unit: ${edanUnit.Name})`, LOG.LS.eTEST);
+        //         return false;
+        //     }
+
+        //     LOG.info(`Created new unit in Packrat (id: ${packratUnit.idUnit}} | unit: ${packratUnit.Name})`, LOG.LS.eTEST);
+        //     return true;
+        // }
+
+        // copy properties over. skip the id or things will break
+        packratUnit.Name = edanUnit.Name;
+        packratUnit.Abbreviation = edanUnit.Abbreviation;
+        packratUnit.ARKPrefix = edanUnit.ARKPrefix;
+
+        const result: boolean =  await packratUnit.update();

```

Comment:
This does not look correct.  We never want to update records in the Unit table based on Edan Verification.  Perhaps we want to add records in the UnitEdan table, when Edan says that it's notion of unit doesn't match anything that we have ... but that's separate code.